docker-compose.yml

```diff
@@ -68,7 +68,7 @@ services:
       - START_BLOCK_STEP=32
       # with 8 nodes you can process up to 4096 blocks at once but rather keep it at 1024 if server doesn't have a lot of RAM.
       # Currently we support node with 16Gb of RAM. To change the capacity go in `./crawler/package.json` and change --max-old-space-size=16384 to desired size
-      - MAX_BLOCKS_PER_STEP=32
```

Comment:
why has this been increased?

Reply:
when crawling data on localhost I prefer using a bigger step for less frontend lag